scripts/validate_min_versions_in_sync.py

```diff
@@ -57,29 +60,76 @@ def get_versions_from_ci(content: list[str]) -> tuple[dict[str, str], dict[str,
             seen_required = True
         elif "# optional dependencies" in line:
             seen_optional = True
+        elif "pip" in line:
```

Comment:
just in case some package with `pip` as part of its name were added to the dependencies
```suggestion
        elif "- pip:" in line:
```

Reply:
Yes that's a good point. Thanks for that.